install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/values.yaml

```diff
@@ -11,3 +11,6 @@ nodeSelector: {}
 # health check to redirect request to sidecar. This makes liveness check work
 # even when mTLS is enabled.
 rewriteAppHTTPProbe: false
+
+# If true, the default interception mode with be NONE
+interceptionNoneByDefault: false
```

Comment:
wasn't there a global config for interception mode?

Reply:
instead of adding a new field which goes out of sync with proxyConfig.Interception mode, try reading the proxyConfig.InterceptionMode in the sidecarInjector config map..

Reply:
` proxyConfig.InterceptionMode` does not have NONE as an option so we cannot use this. @costinm wanted to avoid adding a new field here, so I went with this option.